tests/tests_views.py

```diff
@@ -0,0 +1,78 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+CAR_URL = reverse("taxi:car-list")
+DRIVER_URL = reverse("taxi:driver-list")
+
+
+class PublicAccessTests(TestCase):
+    def test_manufacturer_public(self) -> None:
+        response = self.client.get(MANUFACTURER_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_car_public(self) -> None:
+        response = self.client.get(CAR_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_driver_public(self) -> None:
+        response = self.client.get(DRIVER_URL)
+        self.assertNotEqual(response.status_code, 200)
```

Comment:
no need to use "public" for naming here, u already indicated that in class name )

Reply:
indeed, corrected.